amy/templates/fiscal/all_memberships.html

```diff
@@ -41,7 +42,10 @@
           <td>{{ membership.agreement_start|date:"Y-m-d" }} &mdash; {{ membership.agreement_end|date:"Y-m-d" }}</td>
           <td>{{ membership.get_contribution_type_display }}</td>
           <td {% if membership.instructor_training_seats_remaining <= 0 and membership.instructor_training_seats_total > 0 or membership.instructor_training_seats_remaining < 0 and membership.instructor_training_seats_total == 0 %}class="table-danger"{% endif %}>
-            {{ membership.instructor_training_seats_remaining }}
+            {{ membership.instructor_training_seats_remaining }} / {{ membership.instructor_training_seats_total }}
```

Comment:
This notation (`remaining count` / `total count`) is clever, but I'm wondering if users will know this is remaining / total, and not utilized / total.

Reply:
I'll check this with the CT, but as these calculations have been in place for the other seat types, I wouldn't want to change it without consulting them.